packages/jsii-pacmak/lib/targets/dotnet/dotnetgenerator.ts

```diff
@@ -272,9 +272,16 @@ export class DotNetGenerator extends Generator {
             this.code.line();
         }
 
+        if (initializer) {
```

Comment:
would making the null check in the "emitIfNecessary" be cleaner?

Reply:
Sure. will move the check around.